timescaledb/results/c6a.4xlarge.json

```diff
@@ -1,58 +1,232 @@
 {
-    "system": "TimescaleDB",
-    "date": "2022-07-01",
-    "machine": "c6a.4xlarge, 500gb gp2",
-    "cluster_size": 1,
-    "comment": "",
-
-    "tags": ["C", "PostgreSQL compatible", "row-oriented", "time-series"],
-
-    "load_time": 1620,
-    "data_size": 72882392030,
-
-    "result": [
-[437.700,215.793,176.420],
-[327.026,259.568,244.578],
-[262.978,263.090,263.083],
-[262.807,263.046,266.847],
-[337.497,334.964,330.852],
-[355.689,356.801,362.894],
-[262.762,263.012,262.968],
-[263.055,263.016,263.028],
-[319.928,319.388,320.704],
-[323.584,322.224,322.488],
-[265.979,265.465,265.375],
-[266.019,265.543,265.462],
-[277.018,276.300,276.595],
-[280.352,279.251,279.572],
-[279.915,279.896,279.674],
-[296.377,298.506,297.659],
-[314.448,314.605,312.570],
-[302.668,302.672,303.039],
-[325.810,324.061,324.376],
-[262.447,262.698,262.704],
-[267.581,267.467,267.482],
-[268.085,267.466,267.696],
-[263.391,263.097,263.126],
-[38.291,0.435,0.335],
-[0.127,0.005,0.005],
-[263.138,263.100,263.092],
-[0.889,0.341,0.339],
-[267.586,267.498,267.491],
-[289.086,290.012,290.093],
-[263.220,263.071,263.109],
-[274.780,273.995,273.998],
-[282.217,281.390,281.470],
-[429.273,426.588,439.431],
-[448.808,418.724,418.207],
-[455.196,422.750,423.142],
-[299.263,296.937,297.261],
-[18.693,1.552,1.481],
-[18.125,0.945,0.937],
-[18.528,1.062,0.902],
-[18.268,1.779,1.770],
-[19.615,1.965,1.966],
-[18.970,1.435,1.430],
-[18.330,1.153,0.952]
-]
+  "system": "TimescaleDB",
+  "date": "2024-11-12",
+  "machine": "c6a.4xlarge, 500gb gp2",
+  "cluster_size": 1,
+  "comment": "",
+  "tags": [
+    "C",
+    "PostgreSQL compatible",
+    "column-oriented",
+    "time-series"
+  ],
+  "load_time": 2925,
+  "data_size": 19304833024,
+  "result": [
```

Comment:
Could you please retain the previous formatting, I don't know if the script which scrapes the results into the html will survive otherwise.

Reply:
Updated!